README.md

```diff
@@ -4,7 +4,27 @@
 
 Appcircle CLI is a unified tool for accessing the Appcircle platform features from the command line.
 
-# Installation
+## Table of contents
```

Comment:
IMO, this list should have active voice to better user-friendly

for example,

- Table of Contents
- Installation Instructions
- Usage Guidelines
   - Command List
- Environment Variable Settings
- Interactive Mode Details
- Request Logging
- Guides and Tutorials
   - Configuring Your Appcircle CLI 
   - Environment
   - Trusting Your Self-Hosted Appcircle
   - Certificate
   - Connecting Your Appcircle Account via CLI
   - Starting a New Build via the Appcircle CLI
   - Distributing an App via the Appcircle CLI
- How to Contribute
- Publishing Guidelines
- Jenkins Pipeline Integration
- Docker Image Usage